lib/new-admin/graphql/resolvers/transaction.resolver.js

```diff
@@ -36,7 +36,8 @@ const resolvers = {
   },
   Mutation: {
     cancelCashOutTransaction: (...[, { id }]) => cashOutTx.cancel(id),
-    cancelCashInTransaction: (...[, { id }]) => cashInTx.cancel(id)
+    cancelCashInTransaction: (...[, { id }]) => cashInTx.cancel(id),
+    setTesting: (...[, { txClass, id, testing }]) => transactions.setTesting(txClass, id, testing)
```

Comment:
Better to have a mutation for enabling an address as testing and another for disabling. This way we don't rely on a potentially faulty value coming from the front-end (check our testing customer flag on `customer.type.js`)

```suggestion
    enableTestTransaction: (...[, { txClass, id}]) => transactions.enableTestTransaction(txClass, id),
    disableTestTransaction: (...[, { txClass, id}]) => transactions.disableTestTransaction(txClass, id)
```